pkgs/development/python-modules/awscrt/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib, buildPythonPackage, fetchPypi, cmake, gcc10, perl }:
+
+buildPythonPackage rec {
+  version = "0.11.13";
+  pname = "awscrt";
+
+  propagateBuildInputs = [ cmake gcc10 perl ];
```

Comment:
Looks like a typo, but I don't think packages that use `awscrt` need these build tools.